src/tests/electron/common/view-controllers/left-nav-controller.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { needsReviewTestConfig } from 'electron/platform/android/test-configs/needs-review/test-config';
+import { TestConfig } from 'electron/types/test-config';
+import { leftNavAutomationId } from 'electron/views/left-nav/left-nav';
+import { getAutomationIdSelector } from 'tests/common/get-automation-id-selector';
+import { SpectronAsyncClient } from 'tests/electron/common/view-controllers/spectron-async-client';
+
+export class LeftNavController {
+    private leftNavSelector = getAutomationIdSelector(leftNavAutomationId);
+
+    constructor(private readonly client: SpectronAsyncClient) {}
+
+    public async clickNeedsReview(): Promise<void> {
+        await this.clickLeftNavItem(needsReviewTestConfig);
+    }
+
+    private async clickLeftNavItem(config: TestConfig): Promise<void> {
+        const selector = this.getSelectorForLeftNavItemLink(config);
+        await this.client.click(selector);
+    }
+
+    private getSelectorForLeftNavItemLink(config: TestConfig): string {
+        return `${this.leftNavSelector} li [name="${config.contentPageInfo.title}"] a`;
```

Comment:
Like you mentioned in the teams chat, I think it would be preferable to be able to use a data automation selector here for the link itself. I didn't see an obvious way to add an automation id to the `<a>` tag, but it looks like we're already adding them to immediately children of the link (see `TestViewLeftNavLink` along with usage in `AutomatedChecksViewSelectors.nthTestInLeftNav`), which does what we want when clicked; I'd recommend using that (probably updating `TestViewLeftNavLink` to generate `data-automation-id` based on `link.key`, rather than using the same one for each nav link)